packages/create-sitecore-jss/src/templates/nextjs-sxa/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should use our `tsconfig.json` from the `nextjs` initializer, there are all rules